### PR TITLE
WIP : Any number of channels

### PR DESCRIPTION
**Pull request by BadrYoubiIdrissi**
build [Coverage Status](https://coveralls.io/github/BadrYoubiIdrissi/lucid?branch=any_number_of_channels)

Modified the image parametrization to accept any number of channels. 

The following choices were made :
- No decorrelation is done across channels. 
- A sigmoid is applied on all channels. 

On top of the main channels modification, there is a commit to fix a bug switching width and height.

### CODE REVIEW

File: lucid/optvis/param/color.py [hunk 1]
```diff
@@ -73,3 +73,16 @@ def to_valid_rgb(t, decorrelate=False, sigmoid=True):
     return tf.nn.sigmoid(t)
   else:
     return constrain_L_inf(2*t-1)/2 + 0.5
+
+def arbitrary_channels_to_rgb(*args, channels=None, **kwargs):
```
Comment (colah) [inline lucid/optvis/param/color.py]: I'm not sure I understand when I'd use this function in practice. Is it just for the tests? Maybe add a comment. :)

Comment (BadrYoubiIdrissi) [reply lucid/optvis/param/color.py]: Yes exactly :) It's a funky parametrization to test more than 3 channels

Comment (colah) [reply lucid/optvis/param/color.py]: Makes sense! Consider adding a brief comment, or even moving into the tests? Otherwise, LGTM!

**Comment by BadrYoubiIdrissi** [comment]
@colah It's done :) I have moved the `arbitrary_channels_to_rgb` function to the tests as you suggested alongside with an integration level test. I also added the `grayscale_image_rgb` to the param package init for direct access. Thank you for reviewing !
